plugins/zoom-to-fit/src/index.ts

```diff
@@ -198,9 +198,10 @@ export class ZoomToFitControl implements Blockly.IPositionable {
         i = -1;
       }
     }
-
-    this.svgGroup.setAttribute('transform',
-        `translate(${this.left}, ${this.top})`);
+    if (this.svgGroup) {
+      this.svgGroup.setAttribute('transform',
+          `translate(${this.left}, ${this.top})`);
+    }
```

Comment:
Nit: make this a bit  more concise.

```suggestion
      this.svgGroup?.setAttribute('transform',
          `translate(${this.left}, ${this.top})`);
```